projects/angular-intersection/src/lib/services/intersection-future.service.ts

```diff
@@ -4,7 +4,7 @@ import { Injectable } from '@angular/core';
 @Injectable({
   providedIn: 'root'
 })
-export class IntersectionWillEnterService extends IntersectionObserverService {
+export class IntersectionFutureService extends IntersectionObserverService {
   constructor() {
     super('200px', 0.0);
```

Comment:
Maybe interesting to set a global offset that gets used positive here, and negative in IntersectionPastService?